tools/dynamo/verify_dynamo.py

```diff
@@ -51,6 +52,27 @@ def get_cuda_version():
     cuda_str_version = cuda_version.group(1)
     return packaging.version.parse(cuda_str_version)
 
+def get_rocm_version():
+    from torch.utils import cpp_extension
+
+    ROCM_HOME = cpp_extension._find_rocm_home()
+    if not ROCM_HOME:
+        raise VerifyDynamoError("ROCM was not found on the system, please set ROCM_HOME environment variable")
+
+    hipcc = os.path.join(ROCM_HOME, "bin", "hipcc")
+    hip_version_str = (
+        subprocess.check_output([hipcc, "--version"])
+        .strip()
+        .decode(*cpp_extension.SUBPROCESS_DECODE_ARGS)
+    )
+    hip_version = re.search(r"HIP version: (\d+[.]\d+)", hip_version_str)
+
+    if hip_version is None:
+        raise VerifyDynamoError("CUDA version not found in `hipcc --version` output")
```

Comment:
nitpick, change `CUDA` to `HIP` in the `VerifyDynamoError` please

Reply:
Made the suggested change. Thanks @zstreet87